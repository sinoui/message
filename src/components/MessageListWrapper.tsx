import styled from 'styled-components';

const MessageListWrapper = styled.div`
  position: fixed;
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 0 8px;
  top: 16px;
  right: 0;
  left: 0;
  z-index: ${(props) => props.theme.zIndex.snackbar};
  line-height: 1.5;
  font-size: ${(props) => props.theme.typography.body1.fontSize}rem;
  font-family: ${(props) => props.theme.typography.fontFamily};
  color: ${(props) => props.theme.palette.text.primary};
  pointer-events: none;
  box-sizing: border-box;
`;

export default MessageListWrapper;
