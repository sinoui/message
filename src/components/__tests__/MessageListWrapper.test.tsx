import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import TestWrapper from './TestWrapper';
import MessageListWrapper from '../MessageListWrapper';

describe('MessageListWrapper 镜像测试', () => {
  it('渲染MessageListWrapper', () => {
    const tree = renderer
      .create(
        <TestWrapper>
          <MessageListWrapper />
        </TestWrapper>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
