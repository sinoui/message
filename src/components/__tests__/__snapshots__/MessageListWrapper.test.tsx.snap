// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessageListWrapper 镜像测试 渲染MessageListWrapper 1`] = `
.c0 {
  position: fixed;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin: 0 8px;
  top: 16px;
  right: 0;
  left: 0;
  z-index: 2900;
  line-height: 1.5;
  font-size: 1remrem;
  font-family: -apple-system,BlinkMacSystemFont,'Segoe UI','PingFang SC','Hiragino Sans GB','Microsoft YaHei',Roboto,'Noto Sans SC','Helvetica Neue',Helvetica,Arial,sans-serif,'Apple Color Emoji','Segoe UI Emoji','Segoe UI Symbol';
  color: rgba(0,0,0,0.87);
  pointer-events: none;
  box-sizing: border-box;
}

<div
  className="c0"
/>
`;
